internal/information/build_info.go

```diff
@@ -0,0 +1,72 @@
+package information
+
+import (
+	"runtime/debug"
+
+	"github.com/golang/glog"
+)
+
+type BuildInfo struct {
+	commitSha       string
+	architecture    string
+	vcsTime         string
+	vcsType         string
+	operatingSystem string
+	goVersion       string
+}
+
+// GetBuildInfo returns the current Build Information as a BuildInfo type (struct) using a built-in debug package and its ReadBuildInfo() function.
+// Upon error, an empty struct is returned along with a logged error.
+func GetBuildInfo() *BuildInfo {
+	b := new(BuildInfo)
+	info, ok := debug.ReadBuildInfo()
+	if !ok {
+		glog.Fatal("unable to get build info")
+	}
+	b.goVersion = info.GoVersion
+	for _, i := range info.Settings {
+		switch i.Key {
+		case "vcs.time":
```

Comment:
The keys are referenced as literal "magic" strings. This is already acceptable in this case but a good practice is to put this kind of magic strings into constants. Could we change that?